src/components/Accordion/Accordion.stories.tsx

```diff
@@ -5,23 +5,25 @@ import { Accordion } from './Accordion';
 import { chromaticViewports } from '../../util/viewports';
 import Icon from '../Icon';
 import NumberIcon from '../NumberIcon';
-import Text from '../Text';
 
 type Args = React.ComponentProps<typeof Accordion>;
 
 export default {
   title: 'Components/Accordion',
   component: Accordion,
   parameters: {
-    badges: ['1.2'],
+    layout: 'centered',
+    badges: ['intro-1.2', 'current-2.0'],
```

Comment:
This will denote what "version" of EDS a component was added in, and where it is now. Can also include other badges like BETA, etc.